Cubical/HITs/Sn/Properties.agda

```diff
@@ -97,34 +107,28 @@ wedgeConSn zero zero {A = A} hlev f g hom = F , (λ _ → refl) , right
                           (g (loop i))
   F (loop i) (loop j) = helper i j
```

Comment:
Why give helper explicitly? I feel like you could inline it here

Reply:
I usually do it like this, since otherwise I'll have to write out all the implicit arguments (which get even messier after toPathP is applied).